src/test/java/teammates/test/cases/webapi/GetTimeZonesActionTest.java

```diff
@@ -42,10 +42,10 @@ protected void testExecute() throws Exception {
          * e.g. New York observes DST, so the offset is not always UTC-05:00 the entire year.
          * e.g. timezones can change, like Caracas modifying their timezone. This affects the offset as well.
          */
-        assertEquals(8 * 60 * 60, output.getOffsets().get("Asia/Singapore").intValue());
-        assertEquals(-5 * 60 * 60, output.getOffsets().get("America/New_York").intValue());
-        assertEquals(11 * 60 * 60, output.getOffsets().get("Australia/Sydney").intValue());
-        assertEquals(0, output.getOffsets().get("Europe/London").intValue());
+        assertEquals(8 * 60 * 60, output.getOffsets().get("Etc/GMT-8").intValue());
```

Comment:
Could you add the following javadoc above the asserts to explain the funky zone IDs?

> There is a quirk in the ETC/GMT time zones due to the tzdb using POSIX-style signs in the zone names and the output abbreviations.
> POSIX has positive signs west of Greenwich, while we are used to positive signs east of Greenwich in practice.
> For example, TZ='Etc/GMT+8' uses the abbreviation "GMT+8" and corresponds to 8 hours behind UTC (i.e. west of Greenwich) even though many people would expect it to mean 8 hours ahead of UTC (i.e. east of Greenwich; like Singapore or China).
> (adapted from tzdb table comments)

Reply:
Got it.